hi-ml-cpath/testhisto/testhisto/utils/test_plots_utils.py

```diff
@@ -64,7 +64,7 @@ def test_plots_handler_plots_only_desired_plot_options(plot_options: Collection[
     plots_handler.slides_dataset = MagicMock()
 
     n_tiles = 4
-    slide_node = SlideNode(slide_id="1", prob_score=0.5, true_label=1, pred_label=0)
+    slide_node = SlideNode(slide_id="1", gt_prob_score=0.1, pred_prob_score=0.8, true_label=1, pred_label=0)
```

Comment:
for a binary case shouldn't this sum to 1? or are we testing the multiclass case?

Reply:
Definitely!